db/python/tables/sequencing_group.py

```diff
@@ -591,9 +591,10 @@ async def get_sequencing_group_counts_by_month(
             project = row['project']
             month_created: date = row['sg_date'].replace(day=1)
             sg_type = row['type']
+            sg_tech = row['technology']
             num_sg = row['num_sg']
 
-            project_histories[project][month_created][sg_type] = num_sg
+            project_histories[project][month_created][sg_type + ':' + sg_tech] = num_sg
```

Comment:
Type and technology are relatively free-text fields that can be user created, it isn't unforeseeable that they could have a colon in them. It would be good to make this split sentinel a bit less likely to collide. I generally use a triple pipe `|||` as no one in their right mind would put that in as an input value.